spot_ros2_control/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.8)
+project(spot_ros2_control)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# Dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+# ROS 2 control specific dependencies
+find_package(controller_manager REQUIRED)
+# find_package(hardware_interface REQUIRED)
+# find_package(controller_interface REQUIRED)
+# find_package(pluginlib REQUIRED)
+# Messages
+find_package(sensor_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+# Spot URDF files
+find_package(spot_description REQUIRED)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  # the following line skips the linter which checks for copyrights
+  # comment the line when a copyright and license is added to all source files
+  set(ament_cmake_copyright_FOUND TRUE)
+  # the following line skips cpplint (only works in a git repo)
+  # comment the line when this package is in a git repo and when
+  # a copyright and license is added to all source files
+  set(ament_cmake_cpplint_FOUND TRUE)
```

Comment:
These can fight with WUD's clang-tidy linters, you may want to disable these.

Reply:
Yeah this got generated by default when I created the ros package -- i can take that out